torch/quantization/quantize.py

```diff
@@ -221,6 +222,15 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, qconfig_dict=None):
+    r"""Converts a float model to dynamic quantized model. Do dynamic training and output a quantized model.
+    """
+    model.eval()
```

Comment:
why do you need .eval here?

Reply:
Both static and dynamic quant work only for models in eval(). This is to ensure that batch norm is handled correctly.

Reply:
Right: dynamic quantization only works for the inference.